src/utils/build.utils.ts

```diff
@@ -330,16 +334,20 @@ const BuildUtils = {
     name,
     externals,
     installPath,
+    importPath,
     options,
   }: BuildPackageArgs) {
     const buildStartTime = performance.now()
     let buildIteration = 1
 
+    importPath = await resolve(installPath, importPath)
```

Comment:
We're still using webpack 4, but we have webpack 5's `enhanced-resolve`. This resolves imports to an absolute path so webpack 4 can load them.